tasks/install.yml

```diff
@@ -0,0 +1,78 @@
+- name: Update repositories cache and install php packages
+  apt:
+    name: "{{ item }}"
+    update_cache: yes
```

Comment:
Split this into its own task, otherwise it will be updated for each entry. Also, the task will change if just the cache has been updated.

Reply:
done